OpenUpSA/wazimap-ng-ui
# lazy loading data mapper

## Description
* creating the data mapper menu using the `profile_indicator_summary`
* getting the indicator data when the user expands a subcategory

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-484

## How is it tested automatically?
we have gui tests and jest tests covering this feature

## How should a reviewer test it locally
* confirm that data mapper is working as normal
* confirm that choropleth is displayed as normal and the calculated values are correct
* confirm that filtering is working fine for both choropleth and charts

## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
